Core/src/org/sleuthkit/autopsy/textextractors/TikaTextExtractor.java

```diff
@@ -528,28 +546,34 @@ private static int getTimeout(long size) {
      * @param context Instance containing config classes
      */
     @Override
-    public void setExtractionSettings(Lookup context) {    
+    public void setExtractionSettings(Lookup context) {
         if (context != null) {
             List<ProcessTerminator> terminators = new ArrayList<>();
             ImageConfig configInstance = context.lookup(ImageConfig.class);
             if (configInstance != null) {
                 if (Objects.nonNull(configInstance.getOCREnabled())) {
                     this.tesseractOCREnabled = configInstance.getOCREnabled();
                 }
+                if (Objects.nonNull(configInstance.getLimitedOCREnabled())) {
+                    this.limitedOCREnabled = configInstance.getLimitedOCREnabled();
+                }
+                else {
+                    System.out.println("NULL SETTING");
```

Comment:
@wschaeferB What is the idea with System.out.println("NULL SETTING") ?

Reply:
@eugene7646 me messing up and leaving in conditionals I was using to debug when I didn't have it working because I had the file size mistakenly set to 1 GB

Thank you for catching it and sorry you had to, I will get it removed asap.